src/middleware/express/login-success.js

```diff
@@ -4,7 +4,7 @@
 
 export default function loginSuccess(options = {}) {
   return function(req, res, next) {
-    if (options.successRedirect !== undefined) {      
+    if (options.successRedirect !== undefined && options.successRedirect !== null) {      
```

Comment:
I can fix this really quickly here @marshallswain unless you've already made the change I suggested.